FSharp.Directory.Build.props

```diff
@@ -16,7 +16,9 @@
   <!-- nuget -->
   <PropertyGroup>
     <!-- default NuGet package restore location -->
-    <NugetPackageRoot Condition="'$(NugetPackageRoot)' == ''">$(UserProfile)\.nuget\packages\</NugetPackageRoot>
+    <NuGetPackageRoot Condition="'$(NuGetPackageRoot)' == ''">$(NUGET_PACKAGES)</NuGetPackageRoot>
```

Comment:
Looks like Roslyn has something to ensure this has a trailing slash, and from your build failure, you need it too.